config/log/server.properties

```diff
@@ -66,6 +66,7 @@ log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
 #log4j.logger.proactive.communication.requests = DEBUG
 #log4j.logger.proactive.dataspaces = DEBUG
 #log4j.logger.proactive.vfsprovider = DEBUG
+#log4j.logger.org.ow2.proactive.scheduler.core.helpers.TableSizeMonitorRunner = DEBUG
```

Comment:
You have this great description in the github conversation, what happens if you uncomment this line. Maybe you can add it here.